### ISSUE TITLE
Settings won't change

### ISSUE THREAD

**Issue by 5hay** [jellyfin/jellyfin-plugin-kodisyncqueue#29]
I have Jellyfin 10.6.2 (official Docker image, Ubuntu 20.04 LTS) with Kodi Sync Queue v5.0 installed.

No matter what I do, the settings won't change at all. I can't disable the plugin, nor change the retention days nor set the items that should be tracked.  The toast message "Settings saved." shows up though.

Any tips? Can't see anything suspicious in the Jellyfin logs.

**Comment by mammo0** [comment]
This doesn't work for me either...

I also tried to create the plugin config file manually in `<jellyfin_data_dir>/plugins/configurations/Jellyfin.Plugin.KodiSyncQueue.xml`:

```xml
<?xml version="1.0"?>
<PluginConfiguration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <txtRetDays>0</txtRetDays>
  <chkEnabled>true</chkEnabled>
  <tkMovies>true</tkMovies>
  <tkBoxSets>true</tkBoxSets>
  <tkTVShows>true</tkTVShows>
  <tkMusic>true</tkMusic>
  <tkMusicVideos>true</tkMusicVideos>
</PluginConfiguration>
```

The file template is from: https://github.com/jellyfin/jellyfin-plugin-kodisyncqueue/issues/15#issuecomment-554661073

But it still doesn't work. If I set `chkEnabled` to `false` the plugin is still started on Jellyfin server start:
```
[2020-09-26 14:27:58.543 +00:00] [INF] [1] Jellyfin.Plugin.KodiSyncQueue.Data.DbRepo: Jellyfin.Plugin.KodiSyncQueue IS NOW STARTING!!!
```

**Comment by oddstr13** [comment]
Fixed in #31
